.github/workflows/ci.yml

```
@@ -0,0 +1,116 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  testsuite:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.2', '7.4']
```

```suggestion
        php-version: ['7.2', '7.4', '8.0']
```
We should add 8, right?